it/mail/chrome/messenger/am-smime.properties

```diff
@@ -1,20 +1,20 @@
+NoSenderEncryptionCert=Hai specificato di crittografare questo messaggio, ma l'applicazione non ha potuto trovare il certificato di crittografia specificato nelle tue impostazioni account E-mail e Newsgroup, oppure il certificato è scaduto.
+MissingRecipientEncryptionCert=Hai specificato di crittografare questo messaggio, ma l'applicazione non ha potuto trovare un certificato di crittografia per %S.
+SignNoSenderEncryptionCert=Hai specificato di firmare digitalmente questo messaggio, ma l'applicazione non ha potuto trovare il certificato di crittografia da includere nel messaggio firmato, o il certificato è scaduto.
+ErrorCanNotEncrypt=Impossibile crittografare il messaggio. Controlla di possedere un certificato e-mail valido per ogni destinatario. Controlla anche che i certificati specificati nelle impostazioni account E-mail e Newsgroup siano validi e affidabili.
+ErrorCanNotSign=Impossibile firmare il messaggio. Controlla che i certificati specificati nelle impostazioni account E-mail e Newsgroup per questo account e-mail siano validi e affidabili.
 
 ## Strings used for in the prefs.
 prefPanel-smime=Sicurezza
-NoSigningCert=Gestione certificati non riesce ad individuare un certificato valido per la firma digitale dei propri messaggi.
-NoEncryptionCert=Gestione certificati non riesce ad individuare un certificato valido che gli altri possano usare per inviarci messaggi di posta cifrati.
+NoSigningCert=Il Gestore certificati non riesce ad individuare un certificato valido che può essere utilizzato per firmare digitalmente i tuoi messaggi.
+NoEncryptionCert=Il Gestore certificati non riesce ad individuare un certificato valido che gli altri possano utilizzare per inviarti messaggi e-mail crittografati.
 
-encryption_needCertWantSame=È necessario specificare un certificato che gli altri possano usare quando inviano qui messaggi cifrati. Si vuole usare lo stesso certificato per cifrare e decifrare i messaggi ricevuti?
-encryption_wantSame=Usare lo stesso certificato per cifrare e decifrare i messaggi ricevuti?
-encryption_needCertWantToSelect=È necessario specificare un certificato che gli altri possano usare per rispondere con messaggi cifrati. Configurare un certificato di cifratura adesso?
-signing_needCertWantSame=Si deve specificare anche un certificato da usare per la firma digitale dei propri messaggi. Usare lo stesso certificato per firmare digitalmente i propri messaggi?
-signing_wantSame=Usare lo stesso certificato per firmare i propri messaggi?
-signing_needCertWantToSelect=Si deve specificare anche un certificato per la firma digitale dei propri messaggi. Configurare un certificato per firmare i messaggi digitalmente?
+encryption_needCertWantSame=Dovresti anche specificare un certificato che gli altri possano utilizzare quando t'inviano messaggi crittografati. Vuoi utilizzare lo stesso certificato per crittografare e decifrare i messaggi inviati a te?
```

Comment:
Like the previous one, the correct entry here is "ti inviano". 
Best, 
Andrea 
Via InoMail on Android-------- Messaggio originale --------24 agosto 2016 03:58:28 CEST, Sherman Dickman <notifications@github.com> ha scritto:
In it/mail/chrome/messenger/am-smime.properties:
>  
> -encryption_needCertWantSame=È necessario specificare un certificato che gli altri possano usare quando inviano qui messaggi cifrati. Si vuole usare lo stesso certificato per cifrare e decifrare i messaggi ricevuti?
> -encryption_wantSame=Usare lo stesso certificato per cifrare e decifrare i messaggi ricevuti?
> -encryption_needCertWantToSelect=È necessario specificare un certificato che gli altri possano usare per rispondere con messaggi cifrati. Configurare un certificato di cifratura adesso?
> -signing_needCertWantSame=Si deve specificare anche un certificato da usare per la firma digitale dei propri messaggi. Usare lo stesso certificato per firmare digitalmente i propri messaggi?
> -signing_wantSame=Usare lo stesso certificato per firmare i propri messaggi?
> -signing_needCertWantToSelect=Si deve specificare anche un certificato per la firma digitale dei propri messaggi. Configurare un certificato per firmare i messaggi digitalmente?
> +encryption_needCertWantSame=Dovresti anche specificare un certificato che gli altri possano utilizzare quando t'inviano messaggi crittografati. Vuoi utilizzare lo stesso certificato per crittografare e decifrare i messaggi inviati a te?

@fenderman79 again ti inviano or tiinviano?

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/postbox/localizations","title":"postbox/localizations","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/postbox/localizations"}},"updates":{"snippets":[{"icon":"PERSON","message":"@shermandickman in #26: @fenderman79 again ti inviano or tiinviano?"}],"action":{"name":"View Pull Request","url":"https://github.com/postbox/localizations/pull/26/files/63340a85f6ae3d0662a2998ad4efa42b76afe2a3#r75983874"}}}